File: College-Sports-WebApp/Database/Models/Competitor.cs
using System.Collections.Generic; 
namespace College_Sports_WebApp.Database.Models{ 

    public class Competitor
    {
        public string id { get; set; }
        public string uid { get; set; }
        public string type { get; set; }
        public int order { get; set; }
        public string homeAway { get; set; }
        public Team team { get; set; }
        public string score { get; set; }
        public List<Linescore> linescores { get; set; }
        public List<Statistic> statistics { get; set; }
        public List<Leader> leaders { get; set; }
        public CuratedRank curatedRank { get; set; }
        public List<Record> records { get; set; }
    }

}

File: College-Sports-WebApp/ClientApp/src/api/models/competitor.ts
/* tslint:disable */
/* eslint-disable */
import { CuratedRank } from '../models/curated-rank';
import { Leader } from '../models/leader';
import { Linescore } from '../models/linescore';
import { Record } from '../models/record';
import { Statistic } from '../models/statistic';
import { Team } from '../models/team';
export interface Competitor {
  curatedRank?: CuratedRank;
  homeAway?: string | null;
  id?: number;
  leaders?: Array<Leader> | null;
  linescores?: Array<Linescore> | null;
  order?: number;
  records?: Array<Record> | null;
  score?: string | null;
  statistics?: Array<Statistic> | null;
  team?: Team;
  type?: string | null;
  uid?: string | null;
}
